src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -487,24 +488,38 @@ class TestEditDescribeListed(BaseTestEditDescribe, L10nTestsMixin):
     __test__ = True
 
     def test_edit_categories_add(self):
-        assert [c.id for c in self.get_addon().all_categories] == [22]
-        self.cat_initial['categories'] = [22, 1]
```

Comment:
Here and below I took the opportunity to clean up and remove hardcoded IDs in tests